documentation/jetty-documentation/src/main/asciidoc/operations-guide/server/server-logging-server.adoc

```diff
@@ -14,94 +14,86 @@
 
-Thanks to the SLF4J library, the logging of the Jetty server can therefore be directed to the implementation (called SLF4J _binding_) of your choice.
+Jetty provides a default binding via the `jetty-slf4j-impl` Maven artifact, but you can plug Jetty into the logging framework (or SLF4J _binding_) xref:og-server-logging-server-custom[of your choice].
 
-The Jetty project provides an SLF4J binding (via the `jetty-slf4j-impl` Maven artifact) that is used as the default SLF4J binding.
+Jetty's server logging is enabled by default with the `logging` Jetty module.
+You typically won't have to enable the `logging` module directly, since it is a transitive dependency of the `server` module, and thus pulled in by many of the most commonly used modules.
 
-The logging of the Jetty server itself is enabled by default with the `logging` Jetty module, which is a transitive dependency of the `server` module and therefore it is typically always enabled.
-
-The `logging` Jetty module is a _virtual_ module (see xref:og-modules-names[this section]) and its default implementation is provided by the `logging-jetty` Jetty module, which uses the Jetty SLF4J binding.
+The `logging` module is a xref:og-modules-names[_virtual_ module] and its default implementation is provided by the `logging-jetty` Jetty module, which uses the Jetty SLF4J binding.
 
 [[og-server-logging-server-default]]
 ====== Default Configuration
 
-The Jetty SLF4J binding is configured with an appender (`org.eclipse.jetty.logging.StdErrAppender`) that directs the logging to `System.err`, and reads its configuration from a file named `jetty-logging.properties` that must be found in the class-path.
-
-The `StdErrAppender` format is:
+Jetty's default SLF4J binding uses an _appender_ (`org.eclipse.jetty.logging.StdErrAppender`) to format a logging message with metadata (like a timestamp) before sending it to `System.err`.
+The default `StdErrAppender` format is:
 
 ----
 <datetime>:<level>:<logger name>:<thread name>:<message>
 ----
 
-where `<datetime>=yyyy-MM-dd HH:mm:ss.SSS`.
+where `<datetime>` is a timestamp with the format `yyyy-MM-dd HH:mm:ss.SSS`.
 
-You can configure `StdErrAppender` by specifying the following properties in `jetty-logging.properties`:
+You can configure the appender via a file named `jetty-logging.properties`, which must be found in the xref:og-start-start-class-path[class-path].
+When you enable the `logging-jetty` module -- either directly or by transitive dependency, as in the following example -- Jetty automatically generates `jetty-logging.properties` for you in `$JETTY_BASE/resources/`:
+
+----
+$ java -jar $JETTY_HOME/start.jar --add-modules=http
+----
 
-org.eclipse.jetty.logging.appender.NAME_CONDENSE=<boolean>::
+[source,subs=quotes,options=nowrap]
+----
+include::jetty[args="--add-module=http",highlight="(logging-jetty *transitively enabled|copy.*\.properties)"]
+----
+
+You can specify the following configuration options in `jetty-logging.properties`:
```

Comment:
I've described a few additional configuration options in this document. All of these were drawn from the default `jetty-logging.properties` generated by the `logging-jetty` module (https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/jetty-home/src/main/resources/modules/logging/jetty/resources/jetty-logging.properties). I may have gotten the descriptions wrong, so please correct any errors I've made.